terragrunt/aws/elasticache/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "elasticache_endpoint" {
+  description = "ElastiCache Endpoint for CRA UPD"
```

Comment:
Not as sensitive as credentials but again care should be take not to deliberately expose it. Just for your awareness.